sdk/eventhub/azure-eventhub/tests/livetest/asynctests/test_reconnect_async.py

```diff
@@ -49,7 +49,8 @@
     receiver = uamqp.ReceiveClient(source, auth=sas_auth, debug=False, timeout=10000, prefetch=10)
     try:
         receiver.open()
-        received.extend([EventData._from_message(x) for x in receiver.receive_message_batch(max_batch_size=2, timeout=10000)])
+        received.extend([EventData._from_message(x) for x in receiver.receive_message_batch(max_batch_size=1, timeout=5000)])
+        received.extend([EventData._from_message(x) for x in receiver.receive_message_batch(max_batch_size=1, timeout=5000)])
```

Comment:
better add some comments here explaining why we're splitting one line to two so that the "future us" won't get confused :)